woocommerce/woocommerce-android
# Add a toggle to set simple and subscription products as virtual

Closes: #10218 


### Description
This PR adds a toggle in the product settings to change the value of `isVirtual` for Simple and Subscription products, this aligns with what Core and iOS does.
User still have the ability to change between "Physical" and "Virtual" product types in the Product type bottom sheet (only for simple products), it'll have the same effect, and it's what iOS is also doing, I don't think we need to make any change there.

### Testing instructions
1. Open a simple product.
2. Open Product Settings.
3. Toggle the "Virtual Product" switch.
4. Go back to the product details, and confirm the change was reflected (the product type row should change its value).
5. Open a subscription product.
6. Open Product Settings.
7. Toggle the "Virtual Product" switch.
8. Go back to the product details, then save.
9. Open the product in wp-admin, and confirm the change was saved.

### Images/gif
<img src="https://github.com/woocommerce/woocommerce-android/assets/1657201/10a62fe6-3b0a-45df-b5ca-488ff17dfd03" width=360 />
